ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/StatementExecutor.java

```diff
@@ -237,6 +238,14 @@ private void handleRunScript(final Command command) {
           ksqlConfig.overrideBreakingConfigsWithOriginalValues(command.getOriginalProperties()),
           overriddenProperties
       );
+      if (QueryCapacityUtil.exceedsPersistentQueryCapacity(
+          ksqlEngine,0, getPersistentQueryLimit(command))) {
+        queryMetadataList.stream()
```

Comment:
move into `private static void terminateQueries(final List<QueryMetadata> queries)` method. 

You can also use `map` like this to make the code more readable:

```
queryMetadataList.stream()
            .filter(q -> q instanceof PersistentQueryMetadata)
            .map(PersistentQueryMetadata.class::cast)
            .map(PersistentQueryMetadata::getQueryId)
            .forEach(queryId -> ksqlEngine.terminateQuery(queryId, false));
```

Finally, do we know/understand what that `false` param to `terminateQuery` does?  I'm assuming the schema registry hasn't been updated at this point, but I'm not sure...